assets/css/_dialogs.scss

```diff
@@ -170,6 +199,78 @@ $dialog-box-default-width: 790px;
   text-align: center;
 }
 
+.map-dialog {
+  width: 100vh;
+  padding: 0;
+}
+
+button.close {
+  z-index: 1002;
+}
+
+.geolocation-input {
+  position: absolute;
+  margin-top: 10px;
+  margin-left: 10px;
+  margin-bottom: 20px;
+  z-index: 1001;
+}
+
+.react-autosuggest__suggestions-container--open {
+  display: block;
+  position: relative;
+  top: 0;
+  width: 214px;
+  border: 1px solid #aaa;
+  background-color: #fff;
+  font-family: ‘Open Sans’, sans-serif;
+  font-weight: 100;
+  font-size: 14px;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 4px;
+  z-index: 2;
```

Comment:
See comment regarding z-index scale.

If this only needs to have any number in order to solve stacking context issues, a z-index of 1 should suffice.